parts/single-layout-wsuwp_uc_project.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Single view layout for a "Project" created through University Center Objects.
+ *
+ * This view is represented as "Team" on nararenewables.org.
+ */
+
+?>
+<section class="row side-right gutter pad-ends">
+	<div class="column one">
+			<?php while ( have_posts() ) : the_post(); ?>
+			<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+
+				<header class="article-header">
+					<hgroup>
+						<h1 class="article-title"><?php the_title(); ?></h1>
+					</hgroup>
+				</header>
+
+				<div class="article-body">
+					<?php the_content(); ?>
+					<?php wp_link_pages( array( 'before' => '<div class="page-links">' . __( 'Pages:', 'spine' ), 'after' => '</div>' ) ); ?>
+				</div>
+
+			</article>
+			<?php endwhile; ?>
+
+	</div>
+	<div class="column two">
+		<h5>Student opportunities</h5>
+		<p><a href="http://stage.nara.wsu.edu/training-opportunities/">Find out more about working with NARA</a></p>
```

Comment:
This link could/should be `<?php echo esc_url( home_url() ); ?>/training-opportunities/` (or made dynamic in some other way).